src/models.ts

```diff
@@ -1,25 +1,21 @@
-import type { RestEndpointMethodTypes } from '@octokit/rest'
+import type { components } from '@octokit/openapi-types'
 import type { Root } from 'mdast'
 import type { ReactElement } from 'react'
 
-export type SemVerGroupTitle = 'breaking changes' | 'features' | 'bug fixes'
+export type SemVerGroup = 'breaking changes' | 'features' | 'bug fixes'
 
-export type MiscGroupTitle = 'others' | 'artifacts' | 'thanks' | 'credits'
+export type MiscGroup = 'others' | 'artifacts' | 'thanks' | 'credits'
 
-export type ReleaseGroupTitle = SemVerGroupTitle | MiscGroupTitle | string
+export type ReleaseGroup = SemVerGroup | MiscGroup | string
 
 export type RepositoryQueryParams = {
   repo: string
   owner: string
 }
 
-export type Repository =
-  RestEndpointMethodTypes['repos']['get']['response']['data']
+export type Repository = components['schemas']['full-repository']
```

Comment:
I'm using this other type instead since it's more straightforward and easier to find for TS, so I can see the final computed type in my IDE.